docs/man/polkit.xml

```diff
@@ -666,8 +666,18 @@ polkit.Result = {
         with the JavaScript filename and line number. Log entries are
         emitted using the <constant>LOG_AUTHPRIV</constant> flag meaning
         that the log entries usually ends up in the file
-        <filename>/var/log/secure</filename>. The
-        <function>log()</function> method is usually only used when
+        <filename>/var/log/secure</filename> or in
+        <command>journalctl -b SYSLOG_FACILITY=10</command> on systemd.
+      </para>
+      <para>
+        On some distributions, the flag <literal>--no-debug</literal> is passed
```

Comment:
Hm, this is what data/polkit.service.in provides by default
https://github.com/polkit-org/polkit/blob/main/data/polkit.service.in#L11

So I find it a bit misleading to call this something distro specific.

Fwiw, I would like to see the `--no-debug` flag dropped and replaced with `--debug`.
The default behavious would be a reasonable non-verbose/non-debug output.
And if you want to have explicit debug output (say for a bug report), one should use `--debug`.

Reply:
When everything works, system services should be silent. We should however add support for LogControl1 to dynamically change the level, I started a branch somewhere but ENOTIME

Reply:
Well, polkitd does produce log messages, but no debug log messages, e.g. I get
```
# journalctl -u polkitd.service
Feb 25 09:16:59 mars polkitd[993]: Started polkitd version 124
Feb 25 09:16:59 mars polkitd[993]: Loading rules from directory /etc/polkit-1/rules.d
Feb 25 09:16:59 mars polkitd[993]: Loading rules from directory /usr/share/polkit-1/rules.d
Feb 25 09:16:59 mars polkitd[993]: Finished loading, compiling and executing 11 rules
Feb 25 09:16:59 mars polkitd[993]: Acquired the name org.freedesktop.PolicyKit1 on the system bus
Feb 25 09:16:59 mars systemd[1]: Started polkit.service - Authorization Manager.
```

I'm not convinced we should dump debug log messages into the journal unless explicitly requested.

Reply:
> When everything works, system services should be silent. We should however add support for LogControl1 to dynamically change the level, I started a branch somewhere but ENOTIME

This isn't my view but I understand this view. :)

Reply:
General information about polkit doesn't seem the right place for information about some binary's options, usage of which is distro-specific, no matter how many distros use it. I like switching polkitd to silent by default though and turning verbose on explicitly. I'll incorporate this idea into my plans for implementing log levels.